python/test/unit/language/test_core.py

```diff
@@ -7332,3 +7332,13 @@ def dtype_tensor_kernel(dtype: tl.constexpr):
         tensor = tl.zeros((1, ), dtype)
 
     dtype_tensor_kernel[(1, )](dtype)
+
+
+@pytest.mark.interpreter
+def test_float_tuple():
+
+    @triton.jit
+    def _namedtuple_float_tuple_kernel():
+        x, y = float('-inf'), float('inf')  # noqa: F841
```

Comment:
```suggestion
        x = float('-inf')
        y = float('inf')
```
To isolate assignment

Reply:
Hello, the duplicate condition needs to be written in one line, in two lines will not trigger the problem, one line is a tuple

Reply:
https://github.com/fla-org/flash-linear-attention/commit/fe0be87278e7611f0101ac44caa13f76bb866276